api.md

```diff
@@ -3139,33 +3139,33 @@ SELECT get_telemetry_report(always_display_report := true);
 
 ---
 
-## hypertable_approximate_row_count() [](hypertable_approximate_row_count)
+## approximate_row_count() [](approximate_row_count)
 
-Get approximate row count for hypertable(s) based on catalog estimates.
+Get approximate row count for hypertable, distributed hypertable, or regular PostgreSQL table based on catalog estimates.
+This function support tables with nested inheritance and declarative partitioning.
 
-#### Optional Arguments [](hypertable_approximate_row_count-optional-arguments)
+The accuracy of approximate_row_count depends on the database having up-to-date statistics about the table or hypertable, which are updated by VACUUM, ANALYZE, and a few DDL commands. If you have auto-vacuum configured on your table or hypertable, or changes to the table are relatively infrequent, you might not need to explicitly ANALYZE your table as shown above. Otherwise, if your table statistics are too out-of-date, running this command will update your statistics and yield more accurate approximation results.
```

Comment:
Sorry, was suggesting this before as follow-on text below the description.  Totally fine to have it before, just now need to "above" -> "below" =)
```suggestion
The accuracy of `approximate_row_count` depends on the database having up-to-date statistics about the table or hypertable, which are updated by VACUUM, ANALYZE, and a few DDL commands. If you have auto-vacuum configured on your table or hypertable, or changes to the table are relatively infrequent, you might not need to explicitly ANALYZE your table as shown below. Otherwise, if your table statistics are too out-of-date, running this command will update your statistics and yield more accurate approximation results.
```

Reply:
Right, sorry for that :)